tests/ping.go

```diff
@@ -98,3 +98,38 @@ func vmiConsoleExpectBatch(vmi *v1.VirtualMachineInstance, expected []expect.Bat
 	}
 	return err
 }
+
+// PingAppJob performs a natcat check (tcp by default) using a pod.
+// Optional port and arguments for the netcat command may be provided.
+// If port is not provided, port "8080" is used.
+func PingAppJob(host, port string, args ...string) error {
+	const (
+		netcat      = "nc"
+		defaultPort = "8080"
```

Comment:
The reason for the default is to express "I do not mind what port is used".
Which is the exact reason this came to be in the first place and a valid scenario IMO.

If we require the user to specify it even though he does not care about it, it may raise the obvious question: Why do you use this port and not something else? Something that is avoidable if we use "nothing".

Reply:
But this is directly connected with the port that is being exposed by a different function, starting the server. You have to care about the port unless there is a different mechanism making sure that we by default expose the port that is expected by this Ping function.

Reply:
> But this is directly connected with the port that is being exposed by a different function, starting the server.

Yes, that is the common use case but not the only one.
If we want to express that we do not care, we use "". The usage in this test uses that as it does not care (because there is no server on the other side).

> You have to care about the port unless there is a different mechanism making sure that we by default expose the port that is expected by this Ping function.

All correct, but I fail to understand why this is bothering you that we support an empty value.

Again, if we explicitly use a port, one may ask why we used a specific one... and in the case we have here, there is no reason at all, it is because we do not care.

If I still did not managed to convince you, I'll change it for team peace.

Reply:
While reading the code, I would ask what service are we pinging and why are we expecting it to succeed. Seeing `testServicePort` passed to this function would help me finding that the same port was previously used in `BeforeEach`. Unless a single module has functions bound together where one sets up the service and another pings it, I'd like to see the port passed and connecting the dependencies.

Reply:
Done